CHANGELOG.md

```diff
@@ -39,7 +39,7 @@ Upgrade guidance:
   `CA_HOST_TARGET_AARCH64_CPU` and `CA_HOST_TARGET_RISCV64_CPU`. The environment variable
   `CA_HOST_TARGET_CPU` remains the same name. Note that `CA_HOST_TARGET_CPU_NATIVE` is no longer
   supported but can be achieved by using `native` as a value for the variants.
-
+* LLVM 16 is no longer supported. 17 and 18 are the only supported versions.
```

Comment:
There is an earlier entry "Support for LLVM versions is now limited to LLVM 16 and LLVM 17. Support for earlier LLVM versions has been removed." This should probably be an edit of that instead, rather than a new entry.

Reply:
good spot